### PR TITLE
Fix isa-l.gyp to source the common sources explicitly in all arches

### PR DESCRIPTION
**Pull request by liranmauda**
### Explain the changes
- Fix isa-l.gyp to source the common sources explicitly in all arches to avoid 
   a situation where we are missing symbols in other arches such as ppc64le

### Issues: Fixed #xxx / Gap #xxx
1. Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=2175612

### CODE REVIEW

File: src/native/third_party/isa-l.gyp [hunk 1]
```diff
@@ -313,11 +313,13 @@
                 'isa-l_crypto/include/',
                 'isa-l_crypto/sha256_mb/',
             ],
+            'sources': [
+                'isa-l_crypto/sha256_mb/sha256_ctx_base.c',
+            ],
```
Comment (guymguym) [inline src/native/third_party/isa-l.gyp:318]: same as for sha1 - isn't there an aliases file for sha256 too that should be added to common sources?

File: src/native/third_party/isa-l.gyp [hunk 2]
```diff
@@ -365,12 +366,14 @@
                 'isa-l_crypto/include/',
                 'isa-l_crypto/sha512_mb/',
             ],
+            'sources': [
+                'isa-l_crypto/sha512_mb/sha512_ctx_base.c',
+            ],
```
Comment (guymguym) [inline src/native/third_party/isa-l.gyp:371]: same question on aliases

**Review by guymguym — COMMENTED** [review]
couple of questions

File: src/native/third_party/isa-l.gyp [hunk 3]
```diff
@@ -255,11 +254,13 @@
                 'isa-l_crypto/include/',
                 'isa-l_crypto/sha1_mb/',
             ],
+            'sources': [
+                'isa-l_crypto/sha1_mb/sha1_ctx_base.c',
+            ],
```
[the remark this answers is not in the document]
Comment (liranmauda) [reply src/native/third_party/isa-l.gyp:259]: I can remove the ref implementation.
trying to compare to https://github.com/noobaa/noobaa-core/pull/7028/commits/02c647c7aa028e7ba4644ea4c77fd0a054378f3e it seems that the alias was in the "else" section for some time 

the aliases moved in https://github.com/noobaa/noobaa-core/pull/6215

WDYT?

[the remark this answers is not in the document]
Comment (liranmauda) [reply src/native/third_party/isa-l.gyp]: Done. 
@guymguym 
do we want to get rid of the ref only in the "else" or also in the `node_arch=="arm64` ? 
another question, do we also want to get rid of: `'isa-l_crypto/sha256_mb/sha256_ref.c',` ?

**Comment by liranmauda** [comment]
Mistakenly converted it to a draft...

File: src/native/third_party/isa-l.gyp (hunk 3 above)
[the remark this answers is not in the document]
Comment (liranmauda) [reply src/native/third_party/isa-l.gyp]: by the look of it `_ref.c` files are used for tests.
Talked to @guymguym and decided to remove them.

**Review by guymguym — APPROVED** [review]

**Pull request merged**